template-sample-app/template/src/data/sources/storesSource.js

```diff
@@ -1,10 +1,10 @@
 const STORES_SOURCE_ID = 'storesSource';
 
-export default {
+const stores = {
```

Comment:
I'm ok with the change.
So I guess that in this case we also want to use 'source' instead of 'stores', right?

Reply:
Yep, my fault

Reply:
Changed